CONTRIBUTING-eng.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to PocketFlow
+[Tencent Opensource project](https://opensource.tencent.com/contribution) encourages all developer's participation。You are welcome to [report Issues](https://github.com/Tencent/PocketFlow/issues) or [pull requests](https://github.com/Tencent/PocketFlow/pulls)。 Before contributing, please read the following.
```

Comment:
Use english period '.' instead of '。' after https://github.com/Tencent/PocketFlow/pulls)

Reply:
Also, use english period '.' instead of '。' after "encourages all developer's participation".